frontend/src/app/teams/components/layout/teams-layout.component.scss

```diff
@@ -1,3 +1,62 @@
+.loadingTop{
+	display: flex; 
+	justify-content: space-between;
+}
+
 .container {
 	height: calc(100vh - 170px);
 }
+
+.teams__error{
+	position: relative;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	min-height: 100vh;
+
+	&-content{
+		z-index: 10;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+	}
+
+	&-h1{
+		color: #d8d7d7;
+		font-size: 380px;
+		font-weight: 600;
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		transform: translate(-50%, -50%);
+		z-index: 5;
+		opacity: 0.25;
+	}
+
+	&-h2{
+		color: #000;
+		font-size: 30px;
+		margin-bottom: 50px;
+		text-align: center;
+		line-height: 1.4;
+	}
+
+	&-button{
+		display: inline-block;
+		padding: 15px 40px;
+		background-color:#068e44;
```

Comment:
use predefined variables for colours. see examples that used $primary coming from "vars". It's for all the colour variables, mostly they are already defined in the vars file.